standalone/test/integration_tests/api/integrationtest_scanner_api.cpp

```diff
@@ -148,6 +149,7 @@ ScannerConfiguration ScannerAPITests::generateScannerConfig(const std::string& h
       .scanRange(DEFAULT_SCAN_RANGE)
       .scanResolution(DEFAULT_SCAN_RESOLUTION)
       .enableIntensities()
+      .enableEncoder(ENCODER_ENABLED)
```

Comment:
You need to include the encoder states in the `ScanDataEqual` matcher in `matcher_and_actions.h` else they are ignored by the `integrationtest_scanner_api` tests.

Reply:
Solved. Code modified to add the encoder states at `ScanDataEqual`. See commit [55257b0](https://github.com/PilzDE/psen_scan_v2/pull/334/commits/55257b0359f5d4db8ab971017f2cea4881df83d2)